src/page-modules/contact/components/form/input.tsx

```diff
@@ -4,13 +4,15 @@ import { PageText, TranslatedString, useTranslation } from '@atb/translations';
 import { andIf } from '@atb/utils/css';
 import { Typo } from '@atb/components/typography';
 import ErrorMessage from './error-message';
-import DescriptionModal from './description-modal';
 import { MonoIcon } from '@atb/components/icon';
 import { Button } from '@atb/components/button';
+import { DescriptionModal } from '.';
 
 type InputProps = {
   label: TranslatedString;
-  description?: string;
+  modalDescription?: string;
```

Comment:
I'm wondering if the modal props should be an object (optional) and get its type from the `DescriptionModalProps`. I think that would make the input's code cleaner.